#!/usr/bin/env python
PACKAGE = "lla2enu"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Crash_threshold", double_t, 1, "Indicates the max distance for which the satus is crash", 1, 0, 2)
gen.add("Unsafe_threshold", double_t, 1, "Indicates the max distance for which the satus is unsafe", 5, 4, 15)

exit(gen.generate(PACKAGE, "client_custom", "parameters"))
